tests/blueprints/test_invoice.py

```
@@ -0,0 +1,291 @@
+import json
+import responses
+import base64
+import json
```

Ruff

Redefinition of unused `json` from line 1

Remove definition: `json`

(F811)